sdk/pinocchio/src/lib.rs

```diff
@@ -1,12 +1,222 @@
-//! A library to build a Solana program in Rust.
+//! # Pinocchio
 //!
-//! This library is intended to be used by on-chain programs only. It provides
-//! a zero-dependency library to minimise dependencies conflits. For off-chain
-//! programs, use instead the [`solana-sdk`] crate, which re-exports all modules
-//! from [`solana-program`].
+//! Pinocchio is a zero-dependency library to create Solana programs in Rust.
+//! It takes advantage of the way SBF loaders serialize the program input parameters
```

Comment:
nit: let's say `SVM loaders` since we're moving away from using the term `SBF` in most places